src/chrome/content/rules/Marketwatch.com.xml

```diff
@@ -1,11 +1,15 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://communitycontent.marketwatch.com/ => https://a248.e.akamai.net/f/1731/67675/12h/communitycontent.marketwatch.com/: (51, "SSL: no alternative certificate subject name matches target host name 'communitycontent.marketwatch.com'")
```

Comment:
I don't understand this. It's complaining about the cert for a248.e.akamai.net not being valid for communitycontent.marketwatch.com. I assume that it isn't so obvious a flaw as that (surely?)